beheerkaart.map

```diff
@@ -0,0 +1,227 @@
+#==============================================================================
+#
+# beheerkaart.map
+# doel: De beheerkaart toont de publieke ruimte uitgesplitst
+# 		naar zakelijk recht. Het laat zien welk zakelijk recht
+# 		de Gemeente Amsterdam heeft op een kadastraal object en of 
+# 		dit recht belast is met opstal en/of erfpacht.
+# 		Zodat duidelijk is voor welk deel van de publieke ruimte
+# 		de Gemeente Amsterdam beheersverantwoordlijkheid draagt.
+#
+#==============================================================================
+
+
+
+MAP
+	NAME                      "beheerkaart"
+	INCLUDE                   "header.inc"
+  
+	WEB
+		METADATA
+			"ows_title"           "beheerkaart"
+			"ows_abstract"        "Zakelijk recht publieke ruimte Amsterdam"
+			"wms_extent"          "100000 450000 150000 500000"
+		END
+	END
+  
+#==============================================================================
+# Gebieden met belast zakelijk recht
+#==============================================================================
+
+	LAYER
+		NAME                    belast
+		GROUP                   bgt
+		INCLUDE                 "connection_dataservices.inc"
+		DATA                    "geometrie FROM (SELECT * FROM public.beheerkaart_basis_kaart WHERE agg_indicatie_belast_recht = true AND objecteindtijd is NULL) as SUBQUERY USING srid=28992 USING UNIQUE identificatie_lokaalid"
```

Comment:
Misschien een commentaartje om `objecteindtijd is NULL` uit te leggen? Dat zie ik de andere .maps niet doen.